RFCs/rfc-0014-data-availability.md

```diff
@@ -0,0 +1,182 @@
+
+Please note that `Actions` and `Reducer` library is also essentially a ZK rollup: Actions are TXs stored on archive nodes. Archive nodes are a direct part of the Mina blockchain, but as they are not constant in block size, they cannot be considered as decentralized as the native Mina chain. As a result, it is a logical approach to say there is not too much difference between using an off chain ZK rollup or the native library. Thus, problems discussed for ZK rollups also extend to the `Actions` and `Reducer` library, even when the library is considered fully functional.
+
+### ZK Rollups and their Problems
+
+ZK rollups are structures designed above blockchains to combine multiple TXs into a single TX and settle it on their settlement layer (a.k.a. a blockchain as the consensus layer). As the process of combining TXs is proved through ZKPs, there is no trust in the settlement process. In regular ZK rollups, the rollup layer does not have its own consensus - they directly utilize the consensus of their settlement layer.
+
+In ZK rollups on Mina, Mina is chosen as the settlement layer. As Mina is already a constant sized ZK L1 blockchain, it is a very suitable choice for ZKP settlement. Plus, o1js supports proof aggregation very well, allowing very high amounts of rollup TXs to be included in a single Mina block.
+
+Rollups are very scalable, fast, and cheap as they divide the cost among multiple TXs. However, they decrease liveness guarantees significantly. Most rollups consist of a single (or a few) server (called the sequencer) responsible for the settlement. If the server becomes unreachable for a reason, then the entire access to the zkApp is censored.
+
+Moreover, even though sequencers do not have the power to change a TX they receive in any way, they decide on the ordering of TXs happening in the same settlement block (thus the name “sequencer”). In financial applications, this creates a problem of arbitrage: Sequencers may put their TXs before some significant buy / sell order to gain considerable amounts of money. In a way, sequencers can see the near future and act accordingly.
+
+The most efficient way of increasing the liveness of a rollup is to include a DA layer in the process. If users submit their TXs into a DA layer (through their own light client) and the sequencer is only responsible for submitting them on Mina, then the problem of censorship or arbitrage disappears. As said before, almost all zkApps on Mina are rollups, so integrating a DA layer natively into Mina is very important for zkApp developers.
+
+## Objectives
+
+This RFC is mainly to start a discussion around possible ways of integrating a DA into Mina and get the developer community involvement in the process. There are for sure a lot of unspoken advantages and bottlenecks missing in this RFC, and it is precisely the motivation to raise some discussion points around these issues.
```

Comment:
I agree with this section, but I think you should provide a few more starting points.
- Where to integrate DA? Is this a rollup-only thing, at the core of the Mina protocol or at the zkApp level?
- What are the trade offs with different approaches, especially when it comes to devX/ergonomic and efficiency?